### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by kily-wmz** [facebookresearch/audiobox-aesthetics#13]
hi！
Your work is amazing! Could you please share the complete training code?

**Comment by androstj** [comment]
Hi, thank you for your interest. Currently, we do not have plans to release the training code.
However, the training loop and loss function are straightforward and described in detail in the paper. You could write your own training script and initialize it the model structure using this repo or finetune on top of our provided checkpoint.
